lib/navigation/route.dart

```diff
@@ -1,13 +1,16 @@
 import 'package:go_router/go_router.dart';
 import 'package:injectable/injectable.dart';
+import 'package:survey_flutter_ic/model/survey_model.dart';
 import 'package:survey_flutter_ic/ui/home/home_screen.dart';
 import 'package:survey_flutter_ic/ui/signin/sign_in_screen.dart';
 import 'package:survey_flutter_ic/ui/splash/splash_screen.dart';
+import 'package:survey_flutter_ic/ui/survey_detail/survey_detail_screen.dart';
 
 enum RoutePath {
   root('/', '/'),
   home('/home', 'home'),
-  signIn('/sign_in', 'sign_in');
+  signIn('/sign_in', 'sign_in'),
+  detail('survey-detail', 'survey-detail');
```

Comment:
we have to update the path enum correspondingly
```suggestion
  surveyDetail('survey-detail', 'survey-detail');
```